layout.css

```diff
@@ -79,44 +79,103 @@
 @media screen and (min-width: 768px) {

```

Comment:
while having two media queries will work, the better way would be to have your page styled for mobile view by default. and then have the media query change the few things that need to be changed to make it desktop sized. it will make your code much dryer. right now the desktop styles dont seem to be right yet. resubmit for full points.